### DISCUSSION TITLE
Does net-lb-app-ext module support multiple frontends?

### DISCUSSION THREAD

**Discussion by Avinash-jayakumar** [GoogleCloudPlatform/cloud-foundation-fabric#1670] [Q&A]
@ludoo @juliocc does the net-lb-app-ext module support the creation of multiple frontends. I am trying to create 2 frontends with different certificates. I dont see an example on how to do it with this module. is that possible in this module ?

**Comment by ludoo** [comment]
I'm not entirely clear what you mean with frontends. In general, the module supports everything that you can do via console/gcloud.
